src/System.Text.Http.Parser/HttpParser.cs

```diff
@@ -581,17 +589,17 @@ private static unsafe bool Contains(byte* searchSpace, int length, byte value)
         }
 
         [MethodImpl(MethodImplOptions.NoInlining)]
-        private static bool TryGetNewLineSpan(ref ReadableBuffer buffer, ref Span<byte> span, out ReadCursor end)
+        private static Span<byte> TryGetNewLineSpan(ref ReadableBuffer buffer, out ReadCursor end)
```

Comment:
ref/out spans are not allowed. 
Not sure if more efficient, I'd guess it is the same.

---
In reply to: [110497909](https://github.com/dotnet/corefxlab/pull/1454#discussion_r110497909) [](ancestors = 110497909)